pipeline: outputs: cloudwatch: Add missing `log_group_class` (#1685)
fluent/fluent-bit-docs: pipeline/outputs/cloudwatch.md

| auto\_create\_group   | Automatically create the log group. Valid values are "true" or "false" (case insensitive). Defaults to false.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |

> | log\_group\_class     | An optional parameter that can be used to specify the log storage class for new log groups, (created when `auto_create_group` is `true`). The storage class of pre-existing log groups can not be modified. Valid values are `STANDARD` (default) and `INFREQUENT_ACCESS`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |

Reviewer: ```suggestion
| log\_group\_class     | Optional. Specifies the log storage class for new log groups when `auto_create_group` is set to `true`. You can't modify the storage class of existing log groups. Valid values are `STANDARD` and `INFREQUENT_ACCESS`. Default: `STANDARD`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
